DataCapturing/DataCapturingService.swift

```diff
@@ -480,6 +480,15 @@ public class DataCapturingService: NSObject {
 
         cacheSynchronizationQueue.async(flags: .barrier) {
             self.persistenceLayer.syncSave(locations: self.locationsCache, accelerations: self.accelerationsCache, toMeasurement: measurement)
+            self.persistenceLayer.load(measurementIdentifiedBy: measurement.identifier) { (measurement) in
+                let accelerationsFile = AccelerationsFile()
+                do {
+                    debugPrint("Writing \(measurement.accelerations.count) accelerations to file.")
+                    try accelerationsFile.append(accelerations: measurement.accelerations, to: measurement.identifier)
+                } catch {
+                    fatalError("Unable to write data to file due to \(error).")
+                }
+            }
```

Comment:
This is the method that saves the measured data from cache to disk. I just added the code to actually write the accelerometer data to file. Of course it also still writes all this data to the database. I am going to remove this in one of the next PRs.